Application/Frontend/components/Friends-Tab/FriendsTab.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Avatar, Group, Text, TextInput, TextInputProps, ActionIcon, useMantineTheme, rem } from '@mantine/core';
+import { IconSearch, IconArrowRight } from '@tabler/icons-react';
+
+export function FriendsTab(props: TextInputProps) {
+  const theme = useMantineTheme();
+
+  return (
+    <div>
+        {/** Logo*/}
+        <div style={{ marginTop: '10px', textAlign: 'center'}}>
+            <Text
+                variant="gradient"
+                fw={500}
+                className="text-xl"
+                component="span"
+                gradient={{ from: "pink", to: "yellow" }}
+                style={{ fontSize: '30px' }} 
+            >
+                Accord
+            </Text>
+        </div>
+
+        {/** Search Box*/}
+        <TextInput
```

Comment:
Great usage of this `TextInput` for a search bar. We will link this to a database query later on where we pass in the user-entered username and use that for our query.